test/fixtures/index.ts

```diff
@@ -19,3 +24,32 @@ export const exampleWordlist: WordList = [
   ["ȻSE", 1925],
   ["I", 1884],
 ];
+
+export const testStoredProjectData = {
+  id: 1,
+  langName: "English",
+  bcp47Tag: "en",
+  authorName: "UnknownAuthor",
+} as StoredProjectData;
+
+export const responseBody = {
+  languages: [
+    { id: "km", name: "Khmer (Battambang Khmer)", keyboards: [Array] },
+    {
+      id: "kxm",
+      name: "Khmer, Northern (Khmer Lue)",
+      keyboards: [Array],
+    },
+    { id: "kdt", name: "Kuay (Old Khmer)", keyboards: [Array] },
```

Comment:
Please include the actually contents from the response (did you copy/paste from a `console.log`? `console.log` omits information deep within objects :/)

```suggestion
    { id: "km", name: "Khmer (Battambang Khmer)", keyboards:  [
                "basic_kbdkhmr",
                "basic_kbdkni",
                "kbdkhmr",
                "khmer10",
                "khmer_angkor",
                "sil_khmer"
            ]
    },
    {
      id: "kxm",
      name: "Khmer, Northern (Khmer Lue)",
      keyboards: [
                "sil_khmer"
            ],
    },
    { id: "kdt", name: "Kuay (Old Khmer)", keyboards: [
                "sil_khmer"
            ]
     },
```

(this comes from <https://api.keyman.com/search?q=l:Khmer>)